.ci/deploy-linux.sh

```diff
@@ -43,14 +44,15 @@ if [ "$DEPLOY_APPIMAGE" = "true" ]; then
 
     mv ./RPCS3*.AppImage "$RPCS3_APPIMAGE"
 
-    # If we're building using Azure Pipelines, let's copy over the AppImage artifact
+    # If we're building using a CI, let's copy over the AppImage artifact
     if [ -n "$BUILD_ARTIFACTSTAGINGDIRECTORY" ]; then
-        cp "$RPCS3_APPIMAGE" ~/artifacts
+        mkdir ../artifacts
```

Comment:
why not use ARTDIR here?
or is it something else ?

Reply:
If I don’t create this dir cp renames the file.